extensions/amp-list/amp-list.md

```diff
@@ -261,6 +261,35 @@ We recommend using `binding="no"` or `binding="refresh"` for faster performance.
 
 If `binding` attribute is not provided, default is `always`.
 
+## Experimental: Infinite Scroll (amp-list-load-more)
```

Comment:
A google doc version of this has been shared internally at go/amp-list-load-more-doc for easier iteration.

Reply:
/cc @CrystalFaith

Reply:
Filed follow up https://github.com/ampproject/amphtml/issues/19502 assigned to @CrystalFaith 
 and myself.